dabapush/Reader/TwacapicReader.py

```diff
@@ -25,64 +31,64 @@ def __init__(self, config: 'TwacapicReaderConfiguration'):
         """
         super().__init__(config)
 
-    def unpack_tweet(
-            self,
-            tweet: Dict,
-            includes: Dict,
-            keys: List[str] = ["media", "user", "entities.mentions"]
-        ):
+    def unpack_tweet(  # pylint: disable=W0102
+        self,
+        tweet: Dict,
+        includes: Dict,
+        keys: List[str] = ["media", "user", "entities.mentions"],
+    ):
+        """unpack tweets"""
 
-        possible_keys = ["media","user","entities.mentions"]
+        possible_keys = ["media", "user", "entities.mentions"]
```

Comment:
please put fixes to formatting in their own commits next time

Reply:
Automatically done by black at commit-time, so we'll have to live with that or write like black from the get-go.

Reply:
that's another reason not to do it automatically. Really, it makes reviewing PRs a PIA

Reply:
Nevertheless, if you blacken the code right away before working on it, you can still place it in its own commit. Or cherry pick lines, or whatever